apps/public-docsite-v9/src/DocsComponents/DirSwitch.stories.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+import { makeStyles, Label, Switch, tokens, useId, typographyStyles, shorthands } from '@fluentui/react-components';
+import type { SwitchProps } from '@fluentui/react-components';
+import { DIR_ID } from '@fluentui/react-storybook-addon';
+import addons from '@storybook/addons';
+
+const useStyles = makeStyles({
+  container: {
+    alignItems: 'center',
+    columnGap: tokens.spacingHorizontalXS,
+    display: 'flex',
+    justifyContent: 'start',
+    ...shorthands.margin(tokens.spacingVerticalS, 0),
+  },
+  label: {
+    ...typographyStyles.subtitle2,
+  },
+});
+
+/**
+ * Dir switch used in the react-components docs header
+ */
+export const DirSwitch: React.FC<{ dir?: 'ltr' | 'rtl' }> = ({ dir }) => {
+  const switchId = useId('dir-switch');
+
+  const styles = useStyles();
+
+  const [currentDir, setCurrentDir] = React.useState(dir);
+  const checked = currentDir === 'rtl';
+
+  const setGlobalDir = (newDir: 'ltr' | 'rtl'): void => {
+    addons.getChannel().emit('updateGlobals', { globals: { [DIR_ID]: newDir } });
+  };
+
+  const onChange = React.useCallback<NonNullable<SwitchProps['onChange']>>((_, data) => {
+    const newDir = data.checked ? 'rtl' : 'ltr';
+    setGlobalDir(newDir);
+    setCurrentDir(newDir);
+  }, []);
+
+  return (
+    <div className={styles.container}>
```

Comment:
nit: I would render this inline with the theme picker, but it's very personal preference s🤷

Reply:
Done